project-base/build.xml

```diff
@@ -351,6 +358,20 @@
         </exec>
     </target>
 
+    <target name="generate-build-version">
+        <exec executable="${path.php.executable}" checkreturn="true" outputProperty="version">
+            <arg value="-r" />
+            <arg value="echo date('YmdHis');" />
```

Comment:
Isn't that unique version overkill? It's not possible to deploy every second and you will have only one previous version in the same time. So it's not necessary to have that long version.

Reply:
Maybe you're right, but I don't feel the need to rewrite it. 
Can it cause some problems?